static/app/utils/issueTypeConfig/index.tsx

```diff
@@ -44,6 +44,7 @@ const BASE_CONFIG: IssueTypeConfig = {
   autofix: false,
   eventAndUserCounts: {enabled: true},
   events: {enabled: true},
+  logLevel: {enabled: false},
```

Comment:
Just confirming that `group.level` isn't used for other issue platform issues, I don't see them on the issue stream so I assume not, but mentioning just in case.

Reply:
yup, discussed with Rachel in the metric issues sync. thanks for confirming!